redisinsight/ui/src/components/oauth/oauth-social/OAuthSocial.tsx

```diff
@@ -61,25 +63,36 @@ const OAuthSocial = ({ type = OAuthSocialType.Modal }: Props) => {
   ]
 
   const buttons = socialLinks.map(({ icon, label, className = '', onButtonClick }) => (
-    <EuiButtonIcon
+    <EuiToolTip
       key={label}
-      iconType={icon}
-      className={cx(styles.button, className)}
-      onClick={() => {
-        dispatch(signIn())
-        dispatch(setIsAutodiscoverySSO(isAutodiscovery))
-        isAutodiscovery && dispatch(setOAuthCloudSource(OAuthSocialSource.Autodiscovery))
-        onButtonClick()
-      }}
-      data-testid={label}
-      aria-labelledby={label}
-    />
+      position="top"
+      anchorClassName="euiToolTip__btn-disabled"
```

Comment:
anchor will always have "euiToolTip__btn-disabled" class, I think this class should be only if ```!agreement```

Reply:
Good point. thanks